articles/Monerium/monerium.md

```diff
@@ -0,0 +1,155 @@
+* [The over-collateralization percentage shall be 3% instead of 2%](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52020PC0593#:~:text=The%20percentage%20referred%20to%20in%20Article%C2%A031(1)%2C%20point%20(b)%2C%20shall%20be%20set%20at%203%25%20of%20the%20average%20amount%20of%20the%20reserve%20assets%20for%20issuers%20of%20significant%20asset%2Dreferenced%20tokens.)
+* [A plan should be in place to be able to orderly wind down the company's activities.](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52020PC0593#:~:text=Chapter%206-,Orderly%20wind%2Ddown,-Article%2042%0AOrderly)
+
+### Smart Contracts
+
+There were no smart contract audits being provided for Monerium's EURe. However, we decided to analyze the contracts ourselves, specifically looking for centralization vectors and elevated rights.
+
+What we found was the following:
+
+* [The owner](https://etherscan.io/address/0xA176a20E3698bAa36Ea09E4499e407e7dbFdfa0c) of the [EURe contract](https://etherscan.io/address/0x3231cb76718cdef2155fc47b5286d82e6eda273f#writeContract) is an externally owned account (EOA)
+* [The owner](https://etherscan.io/address/0xA176a20E3698bAa36Ea09E4499e407e7dbFdfa0c) of the [EURe contract](https://etherscan.io/address/0x3231cb76718cdef2155fc47b5286d82e6eda273f#writeContract) has the ability to upgrade the controller. (And hence being able to infinitely mint EURe)
+* [The owner](https://etherscan.io/address/0xA176a20E3698bAa36Ea09E4499e407e7dbFdfa0c) of the [EURe contract](https://etherscan.io/address/0x3231cb76718cdef2155fc47b5286d82e6eda273f#writeContract) has the ability to call the `selfdestruct` function, which kills the contract and transfers all EURe to the owner.
+* [A SystemAccount](https://etherscan.io/address/0x882145b1f9764372125861727d7be616c84010ef) (within the controller) is an EOA.
+* [A SystemAccount](https://etherscan.io/address/0x882145b1f9764372125861727d7be616c84010ef) is able to (infinitely) mint EURe.
+* [The controller contract](https://etherscan.io/address/0x914C6a264bC3cC68966C33BBD6630F011E7a47cF) is not verified on etherscan.
+
+Followed by identifying the above, we reached out to the Monerium team to discuss the issues and how they could potentially be resolved. The result of this discussion was satisfactory in the sense that we were promised that all the issues listed above will be taken care of. This includes:
+
+* Replacing the owner of the EURe contract with a multi-sig
+* Keeping the SystemAccount as an EOA (for operational reasons > automating mints/burns), however implementing changes into the controller that would put a limit on how much EURe can be minted. This would greatly reduce the repercussions of a compromised SystemAccount.
```

Comment:
... and there is no way for anyone to check they deploy these changes without a verified controller contract.

Reply:
See comment above 🙌